articles/specs/launch.0.1.1.xsd

```diff
@@ -172,6 +151,29 @@
           </xs:complexType>
         </xs:element>
       </xs:sequence>
+      <xs:attribute name="file" type="xs:string" use="required">
```

Comment:
Out of curiosity, does moving this around have any affect on what is allowed by the schema?

Reply:
Attributes apply to the type of the element, not the element itself (see https://www.w3schools.com/xml/el_attribute.asp). It was invalid XSD.